### PR TITLE
Add Transaction Details to the Transaction List view

### PR DESCRIPTION
**Pull request by alextsg**
image

### CODE REVIEW

File: ui/app/helpers/conversions.util.js [hunk 1]
```diff
@@ -7,31 +8,44 @@ export function hexToDecimal (hexValue) {
...
-  conversionRate,
-}) {
-  return getValueFromWeiHex({
-    value,
-    conversionRate,
-    toCurrency: 'ETH',
-    numberOfDecimals: 6,
-  })
+export function getEthConversionFromWeiHex ({ value, conversionRate, numberOfDecimals = 6 }) {
+  const denominations = [ETH, GWEI, WEI]
```
Comment (danjm) [inline ui/app/helpers/conversions.util.js]: If I am reading this right, it is to ensure that the activity log shows the value associated with each activity in the largest possible denomination. Is that correct?

File: ui/app/components/transaction-activity-log/transaction-activity-log.util.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+              break
+            }
+
+            case GAS_PRICE_PATH: {
+              events.push(eventCreator(TRANSACTION_UPDATED_GAS_EVENT, timestamp, value))
+              break
+            }
+
+            default: {
+              events.push(eventCreator(value, timestamp))
```
Comment (danjm) [inline ui/app/components/transaction-activity-log/transaction-activity-log.util.js]: `events.push(eventCreator(value, timestamp))` 

The `eventCreator(value, timestamp)` call needs some default parameter for the `eventKey`

File: ui/app/helpers/conversions.util.js (hunk 1 above)
Comment (alextsg) [reply ui/app/helpers/conversions.util.js]: Right, or another way of putting it, if the value is too small for a particular denomination (and the result is 0 ETH, for example), it goes to a lower denomination and so on.

Comment (danjm) [reply ui/app/helpers/conversions.util.js]: I think for the "transaction created" row, the value should always be shown in ETH. When the user creates the transaction, they will be creating it in ETH, so showing in GWEI or WEI is confusing.

screenshot from 2018-09-06 15-14-45

**Comment by danjm** [comment]
Styling issue in contract deployment 'sender-to-recipient'

screenshot from 2018-09-06 15-37-05

**Comment by danjm** [comment]
I wonder if contract deployments should have a different creation message than 'Transaction created with a value of 0 WEI.'? I'm not positive about this, but I think it is worth considering.

**Comment by danjm** [comment]
Token transactions should probably show token value for "Amount" in transaction breakdown and for the 'Transaction created' row in the activity log.

screenshot from 2018-09-06 15-41-57

**Comment by danjm** [comment]
The recipient address in the 'sender-to-recipient' for token transactions should probably be the address of the account to which the tokens were transferred, instead of the contract transaction.

**Review by danjm — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**